python/lsst/afw/geom/spanSet.cc

```diff
@@ -0,0 +1,225 @@
+                (ndarray::Array<pixel, 2, 2> (SpanSet::*)(ndarray::Array<pixel, 1, 1> & input) const)
+                &SpanSet::unflatten<pixel, 1>);
+        cls.def("unflatten", (void (SpanSet::*)(ndarray::Array<pixel, 2, 2> & ,
+                                                ndarray::Array<pixel, 1, 1> & ,
+                                                Point2I const &) const) &SpanSet::unflatten<pixel, 2, 1>,
+                                                py::arg("output"),
+                                                py::arg("input"),
+                                                py::arg("xy0") = Point2I());
+    }
+
+    template <typename pixel, typename PyClass>
+    void declareSetMaskMethod(PyClass & cls) {
+        cls.def("setMask", (void (SpanSet::*)(lsst::afw::image::Mask<pixel> &, pixel) const)
+                           &SpanSet::setMask<pixel>);
+    }
+
+    template <typename pixel, typename PyClass>
+    void declareClearMaskMethod(PyClass & cls) {
+        cls.def("clearMask",
+                (void (SpanSet::*)(lsst::afw::image::Mask<pixel> &, pixel) const) &SpanSet::clearMask<pixel>);
+    }
+
+    template <typename pixel, typename PyClass>
+    void declareIntersectMethod(PyClass & cls) {
+        cls.def("intersect", (std::shared_ptr<SpanSet> (SpanSet::*)(lsst::afw::image::Mask<pixel> const &,
+                                                                    pixel const &) const)
+                                                                    &SpanSet::intersect<pixel>);
+    }
+
+    template <typename pixel, typename PyClass>
+    void declareIntersectNotMethod(PyClass & cls) {
+        cls.def("intersectNot", (std::shared_ptr<SpanSet> (SpanSet::*)(lsst::afw::image::Mask<pixel> const &,
+                                                                       pixel const &) const)
+                                                                       &SpanSet::intersectNot<pixel>);
+    }
+
+    template <typename pixel, typename PyClass>
+    void declareUnionMethod(PyClass & cls) {
+        cls.def("union", (std::shared_ptr<SpanSet> (SpanSet::*)(lsst::afw::image::Mask<pixel> const &,
+                                                                pixel const &) const)
+                                                                &SpanSet::union_<pixel>);
+    }
+
+    template <typename pixel>
+    void declareMaskToSpanSetFunction(py::module & mod) {
+        mod.def("maskToSpanSet",
+                []
+                (lsst::afw::image::Mask<pixel> mask)
+                {
+                    return maskToSpanSet(mask);
+                });
+        mod.def("maskToSpanSet",
+                []
+                (lsst::afw::image::Mask<pixel> mask, pixel const & bitmask)
+                {
+                    auto functor = [&bitmask](pixel const & pixval){ return (pixval & bitmask) == bitmask; };
+                    return maskToSpanSet(mask, functor);
+                });
+    }
```

Comment:
No, it does not, the default functor has the functionality to build a SpanSet based on any bit being set, and there is also an additional overload to the function that allows specifying a specific bit pattern to look for as well. There is a ticket to supply a convince function that will not only turn a mask to a SpanSet, but split the SpanSet up into a list of isolated SpanSets, producing exactly the behavior you spoke of.